### PR TITLE
Gitorg.py: Remove unused variable

### PR DESCRIPTION
**Pull request by cypher-naman**
This removes an unused variable 'e' from the code.

fixes https://github.com/coala/community/issues/165

### CODE REVIEW

File: gci/gitorg.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def get_issue(url):
         return
     try:
         issue = repo.get_issue(int(issue_number))
-    except Exception as e:
+    except Exception:
```
Comment (jayvdb) [inline gci/gitorg.py]: log the exception in the next line

**Comment by MacBox7** [comment]
@cypher-naman please do the recommended changes.

File: gci/gitorg.py (hunk 1 above)
Comment (cypher-naman) [reply gci/gitorg.py]: How and where should I log them?

Comment (MacBox7) [reply gci/gitorg.py]: `e` holds the exception. Don't remove it, log it. `logger.error` in the next line does that.

**Pull request merged**